Library/ViewModels/PledgeViewModel.swift

```diff
@@ -54,6 +55,10 @@ public protocol PledgeViewModelInputs {
   func paymentAuthorizationViewControllerDidFinish()
   func pledgeAmountViewControllerDidUpdate(with data: PledgeAmountData)
   func pledgeDisclaimerViewDidTapLearnMore()
+  func riskMessagingFootnoteLabelTapped()
+  func riskMessagingModalDismissedForApplePay()
+  func riskMessagingModalDismissedForStandardPledge()
+  func riskMessagingPledgeConfirmationButtonTapped()
```

Comment:
Yea so this is why we don't want to re-use the `PledgeViewModelType` for `RiskMessagingViewController`, because we're now extending the responsibilities of the `PledgeViewModel` to include actions specific to the `RiskMessagingViewController`, when previously all that `PledgeViewModel` cared about was `PledgeViewController`. Similarly `PledgeViewController` now has a bunch of inputs/outputs it does not need.